brayns/common/PropertyMap.h

```diff
@@ -146,6 +156,7 @@ class PropertyMap
         const boost::any _min;
         const boost::any _max;
         bool _readOnly{false};
+        std::function<void(const Property&)> _cb;
```

Comment:
Another better name? When reading the code in set it's not fully clear what _cb means.